package.json

```diff
@@ -36,7 +39,7 @@
     "@nestjs/testing": "^7.1.1",
     "@types/express": "^4.17.6",
     "@types/jest": "^25.2.3",
-    "@types/node": "^14.0.6",
+    "@types/mongoose": "^5.7.36",
```

Comment:
Was Node types removed?

Reply:
Yes, after some testing I found out it's not needed.